x-pack/legacy/plugins/maps/public/shared/layers/sources/ems_file_source/ems_file_source.js

```diff
@@ -114,7 +130,26 @@ export class EMSFileSource extends AbstractVectorSource {
   }
 
   canFormatFeatureProperties() {
-    return true;
+    //tooltipProperties may be empty, as this was not introduced until 7.2
+    return this._descriptor.tooltipProperties && this._descriptor.tooltipProperties.length > 0;
```

Comment:
default value in createDescriptor (which is used by Constructor) will guarantee tooltipProperties is set to empty array even when undefined in original descriptor so check for this._descriptor.tooltipProperties and comment are unnecessary.